templates/mod/php.conf.erb

```diff
@@ -2,20 +2,12 @@
 # PHP is an HTML-embedded scripting language which attempts to make it
 # easy for developers to write dynamically generated webpages.
 #
-#<IfModule prefork.c>
-#  LoadModule php5_module modules/libphp5.so
-#</IfModule>
-#<IfModule worker.c>
-#  # Use of the "ZTS" build with worker is experimental, and no shared
-#  # modules are supported.
-#  LoadModule php5_module modules/libphp5-zts.so
-#</IfModule>
```

Comment:
??
I'm not sure to understand what you say. This block remove lines that were already commented out. Not changing anything.

In puppetlabs-apache, mod_php still couldn't be installed with mpm_worker, only mpm_prefork and mpm_itk are allowed. This is also the default in Debian packages.

But this doesn't mean that php is not thread-safe. PHP is, but extensions are probably not. See http://php.net/manual/en/faq.installation.php#faq.installation.apache2.